### PR TITLE
Flow start API fixes/tweaks

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/api/tests/test_v2.py [hunk 1]
```diff
@@ -1887,7 +1887,7 @@ def test_flow_starts(self):
                                            contacts=[self.joe.uuid],
                                            groups=[hans_group.uuid],
                                            flow=flow.uuid,
-                                           restart_participants=True,
+                                           restart_participants=False,
```
Comment (rowanseymour) [inline temba/api/tests/test_v2.py]: Default is true so need this test to actually test that the param is being passed

File: temba/api/v2/serializers.py [hunk 2]
```diff
@@ -535,6 +535,7 @@ class FlowStartWriteSerializer(WriteSerializer):
     contacts = fields.ContactField(many=True, required=False)
     groups = fields.ContactGroupField(many=True, required=False)
     urns = fields.URNListField(required=False)
+    restart_participants = serializers.BooleanField(required=False)
```
Comment (rowanseymour) [inline temba/api/v2/serializers.py]: This was missing

File: temba/api/v2/views.py [hunk 3]
```diff
@@ -2374,36 +2376,31 @@ class FlowStartsEndpoint(ListAPIMixin, CreateAPIMixin, BaseAPIView):
             "next": "http://example.com/api/v2/flow_starts.json?cursor=cD0yMDE1LTExLTExKzExJTNBM40NjQlMkIwMCUzRv",
             "previous": null,
             "results": [
-            {
-                "id": 150051,
-                "flow": {
-                    name: "Thrift Shop",
```
Comment (rowanseymour) [inline temba/api/v2/views.py]: Should be valid JSON

File: temba/api/v2/views.py [hunk 4]
```diff
@@ -2420,54 +2417,42 @@ class FlowStartsEndpoint(ListAPIMixin, CreateAPIMixin, BaseAPIView):
...
+            "created_on": "2013-08-19T19:11:21.082Z",
+            "modified_on": "2013-08-19T19:11:21.082Z"
         }
 
     """
     permission = 'api.flowstart_api'
     model = FlowStart
     serializer_class = FlowStartReadSerializer
     write_serializer_class = FlowStartWriteSerializer
-    pagination_class = CreatedOnCursorPagination
```
Comment (rowanseymour) [inline temba/api/v2/views.py]: Assuming we want modified_by ordering so users can poll for status changes

File: temba/api/v2/views.py [hunk 5]
```diff
@@ -2420,54 +2417,42 @@ class FlowStartsEndpoint(ListAPIMixin, CreateAPIMixin, BaseAPIView):
...
-    throttle_scope = 'v2.api'
+    pagination_class = ModifiedOnCursorPagination
 
     def get_queryset(self):
         org = self.request.user.get_org()
-        return FlowStart.objects.filter(flow__org=org, is_active=True).order_by('-modified_on', '-id')
+        return self.model.objects.filter(flow__org=org, is_active=True)
```
Comment (rowanseymour) [inline temba/api/v2/views.py]: Ordering handled by paginator

**Comment by norkans7** [comment]
Yep

**Comment by ericnewcomer** [comment]
yup.

**Pull request merged**